pkg/clustersync/clusterSync.go

```diff
@@ -377,3 +393,18 @@ func findStaleClusterResources(ctx context.Context, dynamicClient dynamic.Interf
 	return needToDelete, nil
 
 }
+
+// find the enabled addons from the cluster labels
+func getEnabledAddons(labelMap map[string]interface{}) map[string]interface{} {
+
+	enabledAddons := make(map[string]interface{}, len(allAddons))
+	for _, addon := range allAddons {
+		addonLabelName := "feature.open-cluster-management.io/addon-" + addon
+		if _, ok := labelMap[addonLabelName]; ok {
+			enabledAddons[addon] = "true"
```

Comment:
Wondering if we should keep the state from the label.
```suggestion
		if addonState, ok := labelMap[addonLabelName]; ok {
			enabledAddons[addon] = addonState
```

Reply:
But the label says "available", We will have to change our query if we want to keep that.

Reply:
Our query is looking for true and false values.

Reply:
ok, let's keep it true/false.